<?xml version="1.0" encoding="UTF-8"?><testrun name="FelixTestConnexionImpossible" project="Felix_java" tests="3" started="0" failures="1" errors="0" ignored="0">
  <testsuite name="vue.FelixTestConnexionImpossible" incomplete="true">
    <failure>java.lang.AssertionError: Probl√®me d'invocation de l'application : null
	at org.junit.Assert.fail(Assert.java:88)
	at vue.FelixTestConnexionImpossible.lanceInstance(FelixTestConnexionImpossible.java:146)
	at vue.FelixTestConnexionImpossible.setUp(FelixTestConnexionImpossible.java:103)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:538)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:760)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:206)

</failure>
    <testcase name="testConnexionSansCamix_adresseEtPortValides" classname="vue.FelixTestConnexionImpossible" incomplete="true"/>
    <testcase name="testConnexionSansCamix_adresseInvalide" classname="vue.FelixTestConnexionImpossible" incomplete="true"/>
    <testcase name="testConnexionSansCamix_portInvalide" classname="vue.FelixTestConnexionImpossible" incomplete="true"/>
  </testsuite>
</testrun>
